.circleci/continue_config.yml

```diff
@@ -497,14 +530,14 @@ workflows:
       - log-config-results:
           filters:
             tags:
-              only: /^(proxyd|ufm-[a-z0-9\-]*|op-[a-z0-9\-]*)\/v.*/
+              only: /^(pms|proxyd|ufm-[a-z0-9\-]*|op-[a-z0-9\-]*)\/v.*/
```

Comment:
can we expand the tag name to `peer-mgmt-service` so its consistent with the code and docker image name?

we can keep the abbreviations in circleci configs, but externally it should be consistent